src/App.css

```diff
@@ -1,15 +1,291 @@
+:root {
+  --blue: #0290ff;
+  --white: #ffff;
+  --gray: rgba(0, 0, 0, 0.05);
+  --dark-gray: gray;
+  --border-color: #e8e8e8;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
-  background-color: white;
+  overflow-x: hidden;
+  background-color: var(--gray);
+}
+
+.books-container {
+  width: 100rem;
+  height: 89.625rem;
+  padding: 9.313rem 3.75rem 7.688rem 6.25rem;
+  background-color: var(--pale-grey);
+}
+
+.left {
+  align-items: center;
+  gap: 2.97rem;
+}
+
+header {
+  background-color: var(--white);
+  height: 80px;
+  box-shadow: rgba(0, 0, 0, 0.1) 0 10px 15px -3px, rgba(0, 0, 0, 0.05) 0 4px 6px -2px;
+}
+
+nav {
+  height: 5.938rem;
+  padding: 20px 0;
+}
+
+.home {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.categories {
+  width: 6.563rem;
+  height: 1rem;
+  margin: 1.125rem 41.5rem 0.688rem 2.563rem;
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+a {
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 0.9rem;
+  color: var(--dark-gray);
+}
+
+a:active {
+  color: var(--dark-gray);
+}
+
+.user {
+  border: 1px solid var(--blue);
+  padding: 10px;
+  border-radius: 50%;
+}
+
+.logo {
+  text-transform: uppercase;
+  font-size: 1.8rem;
+  color: var(--blue);
+  padding-left: 6.25rem;
+}
+
+.center {
+  text-align: center;
+}
+
+.right {
+  padding-right: 35px;
+}
+
+.book {
+  display: grid;
+  grid-template-columns: 2fr 1fr 1fr;
+
+  /* display: flex;
+  justify-content: space-between; */
```

Comment:
- [ ] kindly consider removing the commented lines of codes to make your code base look professional